tests/fixtures/CORPUS_LAYOUT.md

```
@@ -163,10 +163,29 @@ BLESS_MOCK_PIN=1 cargo nextest run --features dev -E 'test(regression_under_mock
 ```
 
+## Per-corpus extensions
+
+Some corpora's source tests run the normalizer under multiple configs
+within the same fixture (e.g. biocommons exercises every input under
+both 3'-shifting and 5'-shifting × cross-boundaries on/off). In that
+case, the corpus's case schema adds `shuffle_direction` and
+`cross_boundaries` fields per row, and its `mock-pin/<axis>.txt`
+format extends to include the config in the pin line:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)